pkg/apis/batch/v1alpha1/job.go

```diff
@@ -70,6 +70,15 @@ type JobSpec struct {
 	// Defaults to 3.
 	// +optional
 	MaxRetry int32 `json:"maxRetry,omitempty" protobuf:"bytes,8,opt,name=maxRetry"`
+
+	// ttlSecondsAfterFinished limits the lifetime of a Job that has finished
+	// execution (either Completed or Failed). If this field is set,
+	// ttlSecondsAfterFinished after the Job finishes, it is eligible to be
+	// automatically deleted. If this field is unset,
+	// the Job won't be automatically deleted. If this field is set to zero,
+	// the Job becomes eligible to be deleted immediately after it finishes.
+	// +optional
+	TTLSecondsAfterFinished *int32 `json:"ttlSecondsAfterFinished,omitempty" protobuf:"varint,9,opt,name=ttlSecondsAfterFinished"`
```

Comment:
I think we should add the validation as it should not be negative

Reply:
validate can be applied in admission controller

Reply:
> validate can be applied in admission controller

yeah thats correct we should add the validation in admission controller

Reply:
IIRC @wangyuqing4 will work on admission controller.

Reply:
@hzxuzhonghu , please confirm with  @wangyuqing4 ; if she is not working this, please feel free to open a PR for that.